packages/vue3/src/layouts/table/TheTbody.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <!-- 
+    Used 'item-key' as 'name' to ensure unique identification as columns are generated with 'name' key in the attrs array
+  -->
+  <Draggable
+    v-if="reorder"
+    handle=".v-list-table__drag"
+    v-model="rowsData"
+    item-key="name"
+    tag="tbody"
+  >
+    <slot name="body-start" />
```

Comment:
This slot won't work here as the TheBody component is not being used directly here. The component we'll be using is `VListTable` and that component should forward the slot to `TheBody`.